src/version/features/UserTour/content.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 
+import { getCurrentThemeColor } from "../../../core/config/styles";
+
+const themeColor = getCurrentThemeColor();
```

Comment:
no, use a material-ui style in the function form

Reply:
It will be done in future

Reply:
Done